.github/workflows/actions.yml

```
@@ -1,43 +1,45 @@
+      - master
 
 jobs:
   tests:
+    name: Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.10', '3.11', '3.12', '3.13']
-    name: Python ${{ matrix.python-version }}
+        python-version: 
```

yamllint

[error] trailing spaces

(trailing-spaces)